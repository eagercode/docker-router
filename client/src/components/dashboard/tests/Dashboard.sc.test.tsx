import * as React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import Div from '../Dashboard.sc';

describe('<Div/>', () => {
    it('styled dashboard div snapshot', () => {
        expect(mount(<Div/>)).toMatchSnapshot();
    })
});