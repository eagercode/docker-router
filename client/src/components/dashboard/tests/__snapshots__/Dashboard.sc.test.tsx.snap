// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Div/> styled dashboard div snapshot 1`] = `
.c0 {
  -webkit-column-count: 4;
  column-count: 4;
  margin: 0 auto;
  max-width: 1200px;
  padding: 5px 10px 0 10px;
}

@media only screen and (max-width:62em) {
  .c0 {
    -webkit-column-count: 3;
    column-count: 3;
  }
}

@media only screen and (max-width:48em) {
  .c0 {
    -webkit-column-count: 2;
    column-count: 2;
  }
}

@media only screen and (max-width:23.5em) {
  .c0 {
    -webkit-column-count: 1;
    column-count: 1;
  }
}

<styled.div>
  <div
    className="c0"
  />
</styled.div>
`;
